test.js

```diff
@@ -167,6 +167,21 @@ import { NDivider } from "naive-ui";
 	t.is(formattedCode, code);
 });
 
+test('works with pug templates in Vue files', (t) => {
+	const code = `<script setup lang="ts">
+import Foo from "@/components";
+</script>
+
+<template lang="pug">
+  Foo
```

Comment:
You can't add initial indent to pug template, because pug sensitive to indent, change `  Foo` to `Foo` fixed the problem.